### PR TITLE
fix: add stop for NetworkingStack

### PR DESCRIPTION
**Pull request by mmsqe**

### CODE REVIEW

**Review by itsdevbear — DISMISSED** [review]
based. 

Unrelated but I am thinking of deprecating the usage of the Geth node as it leaks a bunch of ports. Thinking just our own custom json rpc server but utilize the geth handlers. What do you think?

**Comment by mmsqe** [comment]
> based.
> 
> Unrelated but I am thinking of deprecating the usage of the Geth node as it leaks a bunch of ports. Thinking just our own custom json rpc server but utilize the geth handlers. What do you think?

Sounds good, could also custom domain if need.

**Comment by itsdevbear** [comment]
@mmsqe lmk if ur down to try ripping out the geth node and replacing it with our own custom stack

**Comment by mmsqe** [comment]
> @mmsqe lmk if ur down to try ripping out the geth node and replacing it with our own custom stack

Sure, but I'm not allowed to build docker recently but only can play around app_v2 test with [sim](https://github.com/berachain/polaris/compare/6f6d5617276d5bc0c6ce821be2405ba85c32c6fd...mmsqe:sim?expand=1) test

**Comment by itsdevbear** [comment]
not against merging that if you want to make a PR

**Comment by itsdevbear** [comment]
why does your docker not work? @archbear can you help him out

**Comment by mmsqe** [comment]
> why does your docker not work? @archbear can you help him out

oh sry, it's block by local netskope not related here

**Comment by mmsqe** [comment]
> why does your docker not work? @archbear can you help him out

Seems need inject some cert when build for hive

**Comment by itsdevbear** [comment]
We can probably merge this cause why not @calbera

File: Makefile [hunk 1]
```diff
@@ -18,4 +20,11 @@ codeqlbuild:
...
 # Rule to setup the project. This is a special case because it's not a Mage target.
 setup:
-	@go run magefiles/setup/setup.go
\ No newline at end of file
+	@go run magefiles/setup/setup.go
+
+BINDIR ?= $(GOPATH)/bin
+CURRENT_DIR = $(shell pwd)
+
+test-sim-after-import:
```
Comment (itsdevbear) [inline Makefile:28]: @calbera I'm down to start migrating away from mage tbh, more hassle than its been a value add.

**Review by itsdevbear — APPROVED** [review]

**Review by calbera — APPROVED** [review]

**Pull request merged**